docs/integrations/containers-orchestration/kubernetes.md

```diff
@@ -131,60 +127,55 @@ The monitors are disabled by default. Once you have installed the alerts using t
+  ```sql title="Email Notifications Example"
+  email_notifications_critiical = [
+      {
+        connection_type       = "Email",
+        recipients            = ["abc@example.com"],
+        subject               = "Monitor Alert: {{TriggerType}} on {{Name}}",
+        time_zone             = "PST",
+        message_body          = "Triggered {{TriggerType}} Alert on {{Name}}: {{QueryURL}}",
+          run_for_trigger_types = ["Critical", "ResolvedCritical"]
+      }
+    ]
+  ```
+6. **Install the Alerts**:
+   1. Navigate to the package directory `terraform-sumologic-sumo-logic-monitor/monitor_packages/kubernetes/` and run `terraform init`. This will initialize Terraform and will download the required components.
+   1. Run `terraform plan` to view the monitors which will be created/modified by Terraform.
+   1. Run `terraform apply`.
+7. **Post Installation**. If you haven’t enabled alerts and/or configured notifications through the Terraform procedure outlined above, we highly recommend enabling alerts of interest and configuring each enabled alert to send notifications to other people or services. See Step 4 of [this document](/docs/alerts/monitors#add-a-monitor).
 
 :::note
 There are limits to how many alerts can be enabled - see the [Alerts FAQ](/docs/alerts).
 :::
 
-## Upgrading the Kubernetes App
+## Upgrading the Kubernetes app
 
 The current version of our app is not backwards compatible with older Helm Chart versions. Follow the below steps to upgrade.
-1. Switch to [Admin Mode](/docs/manage/content-sharing/admin-mode/#switch-to-admin-mode). If you are not a `Content Administrator` you may have to contact Administrator within your organization to grant `Manage Content` role to your user.
-1. Go to `Library/Admin Recommended/Sumo Logic Integrations/Kubernetes` and rename the folder (for example Kubernetes-backup-Oct-2023) by clicking on the three button and choosing `Edit`.
-1. Go back to the App Catalog and reinstall the `Kubernetes` App again by following instructions in [Installing the Kubernetes App](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) section. If the tab was already open, you may have to refresh the page.
-   
+
+1. Switch to [Admin Mode](/docs/manage/content-sharing/admin-mode/#switch-to-admin-mode). If you are not a Content Administrator, you may have to contact Administrator within your organization to grant the [Manage Content](/docs/manage/users-roles/roles/role-capabilities/#data-management) role to your user.
+1. Go to `Library/Admin Recommended/Sumo Logic Integrations/Kubernetes` and rename the folder (for example, Kubernetes-backup-Oct-2023) by clicking on the three button and choosing **Edit**.
```

Comment:
Let's add a note that after user renames the folder, it may take around 5min for changes to take effect. So wait for 5min & proceed to the next step after that.

Reply:
@hamoli can you please review?

```suggestion
1. Go to the folder **Library/Admin Recommended/Sumo Logic Integrations/Kubernetes**, click on the three button, choose **Edit**, and rename the folder to your liking (for example, **Kubernetes-backup-Nov-2023**). The renaming may take up to 5 minutes to go into effect.
```

Reply:
LGTM